src/main/resources/org/jenkinsci/plugins/gwt/global/WhitelistItem/help-host.html

```diff
@@ -1,9 +1,12 @@
-Is the IP of a server that is allowed to invoke the plugin. This can be
+Is the domain or IP of a server that is allowed to invoke the plugin. This can be
 <b>CIDR</b> or ranges:
 <ul>
+  <li>WINS_SERVER</li>
```

Comment:
The field is currently validated agains `remoteAddr`:
https://github.com/jenkinsci/generic-webhook-trigger-plugin/blob/master/src/main/java/org/jenkinsci/plugins/gwt/GenericWebHookRequestReceiver.java#L58

And there is also `request.getRemoteHost()`. But I changed it to `addr` because I was not sure if `remoteHost()` is reliable. Just so that we dont try to validate and IP range against a domain.

Anyway, `WIN_SERVER` and `example.com` will never work with current implementation because `remoteAddr` always returns an IP. And I think just supporting IP addresses is good enough.

Reply:
Ah okay, makes sense. Just pushed an update nuking the domain validation, etc.